pkgs/by-name/ga/gapless/package.nix

```diff
@@ -54,7 +54,6 @@ stdenv.mkDerivation (finalAttrs: {
     mainProgram = "g4music";
     homepage = "https://gitlab.gnome.org/neithern/g4music";
     license = licenses.gpl3Only;
-    maintainers = with maintainers; [ magnouvean ];
```

Comment:
Are you going to abandon this package?

Reply:
Yeah, sadly I don't use it anymore due to more fitting alternatives on kde plasma (I think I used gnome when I created this package).